Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -111,84 +334,283 @@ The source generator adds the following entry in the CloudFormation template cor
         "CodeUri": ".",
         "MemorySize": 256,
         "Timeout": 30,
-        "Policies": [
-          "AWSLambdaBasicExecutionRole"
-        ],
         "PackageType": "Zip",
-        "Handler": "TestServerlessApp::TestServerlessApp.Functions_Plus_Generated::Plus",
+        "Handler": "CloudCalculator::CloudCalculator.Functions_Add_Generated::Add",
         "Events": {
           "RootGet": {
-            "Type": "Api",
+            "Type": "HttpApi",
             "Properties": {
-              "Path": "/plus/{x}/{y}",
-              "Method": "GET"
+              "Path": "/add/{x}/{y}",
+              "Method": "GET",
+              "PayloadFormatVersion": "2.0"
+            }
           }
         }
       }
-    }
-  }
+    },
 ```
 
-To see the code that is generated by the source generator turn the verbosity to detailed when executing a build. From the command this 
-is done by using the `--verbosity` switch.
-```
-dotnet build --verbosity detailed
-```
-To change the verbosity in Visual Studio go to Tools -> Options -> Projects and Solutions and adjust the MSBuild verbosity drop down boxes.
+Here is an example of the generated code from the source generator for the `Add` Lambda function. The generated code wraps around the 
+C# method that has the `LambdaFunction` attribute. It takes care of
+configuring the dependency injection, get the parameters from the API Gateway event and invoked the wrapped `LambdaFunction`. This code snippet is hear for 
```

Comment:
"configuring the dependency injection, gets the parameters from the API Gateway event, and invokes the wrapped `LambdaFunction`.
*Typo

Reply:
Fixed